### DISCUSSION TITLE
A few first things I noticed

### DISCUSSION THREAD

**Discussion by Gothdir** [mcmonkeyprojects/SwarmUI#6] [Ideas]
Hey,

just tried swarmui for the first time and I noticed a few things which I would like to see. These things might be in it but I didnt find them yet, so bare with me.

1. I noticed that for me the reuse Wildcard button doesnt seem to work properly. It says "Reuse from selected Image". So my expected usage would be I generate a bunch of images with a random Wildcard seed, I see something I like and want to generate with that seed, I select the image and press the Reuse button but that doesnt seem to work because its reusing the random seed. It only works when I select a specific seed in advance instead of random which makes this way less useful.
![image](https://github.com/mcmonkeyprojects/SwarmUI/assets/153355482/f11aa414-f176-4cc1-85a8-2bf5ad803822)

2.  This is more of a question then a request. Is there a way to use Adetailer to Improve parts of a person? Maybe controlnet can be used for that if so I probably have to dig into those more. EDIT: Found out about the <segment:face> (example) function which works pretty well but could be clearer that it exists. I found it it via google.

**Comment by mcmonkey4eva** [comment]
new commit https://github.com/mcmonkeyprojects/SwarmUI/commit/c7f8b2b2ed3d2064ce03cf93748b651b76a81e01 should handle it properly from now on - the wildcard seed will be in the image metadata for new images, and reuse will grab them properly
